lib/chore/strategies/worker/helpers/ipc.rb

```diff
@@ -75,7 +75,7 @@ def ipc_help
 
       private
 
-      # TODO: do we need this as a optional param
+      # TODO: Decide if we should make this customizable via an optional param
```

Comment:
I think the answer is probably "no" since we haven't needed to do it in years :laughing: